Std/Data/BitVec/Lemmas.lean

```diff
@@ -117,6 +117,21 @@ private theorem lt_two_pow_of_le {x m n : Nat} (lt : x < 2 ^ m) (le : m ≤ n) :
   else
     simp [h]
 
+
+/-! ### msb -/
+
+theorem msb_eq_decide (x : BitVec (Nat.succ w)) : BitVec.msb x = decide (2 ^ w ≤ x.toNat) := by
```

Comment:
```suggestion
theorem msb_eq (x : BitVec (Nat.succ w)) : BitVec.msb x = decide (2 ^ w ≤ x.toNat) := by
```
I think the naming convention is [slightly unclear here](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Naming.20convention/near/421544382), but my preference is that the name stops at `eq` if the RHS is obvious and unambiguous. So this could just be `msb_eq`.

Reply:
I guess, however, in this case we also want `x.msb = x.getMsb 0`, and possibly also `x.msb = x.getLsb (Fin.last w)`, so on second thoughts keep the name as is.

What do you think of adding the other two lemmas as well (and possibly also the corresponding ones for `x.lsb`)? Fine if not now.

Reply:
@bollu, I think you may have committed before seeing my second message. :-)

Reply:
@semorrison Yes, I did. I shall keep the name with the `msb_eq_decide ` on this one, and then add the other two as a follow up PR.